docs/components/section.js

```diff
@@ -1,27 +1,16 @@
-import { useState } from 'react'
-import { Layout, Row, Column, FadeIn } from '@carbonplan/components'
-import Sidenav from './sidenav'
+import { NavSection } from '@carbonplan/layouts'
+import { contents } from './contents'
 
 const Section = ({ children, name }) => {
-  const [expanded, setExpanded] = useState(false)
-
   return (
-    <Layout
-      fade={false}
-      settings={{
-        value: expanded,
-        onClick: () => setExpanded((prev) => !prev),
-      }}
+    <NavSection
+      name={name}
+      menu={{ contents }}
+      title={`${name[0].toUpperCase() + name.slice(1)} – CarbonPlan`}
+      description={'TK'}
```

Comment:
is this TK because this is just a template, so it's assumed that it'll be filled in? an alternative would just be to leave the prop unspecified, which will trigger a warning, but i'm fine either way

Reply:
Nice, I like the idea of relying on the console warnings. Removed in [9baf567](https://github.com/carbonplan/create-sphinx-next/pull/4/commits/9baf567161ef4505c960a2fbfc3ec87e9e49493c).